src/main/java/com/spotify/logging/LoggingConfigurator.java

```diff
@@ -121,15 +137,18 @@ public static void configureDefaults(final String ident) {
    *
    * @param ident The logging identity.
    * @param level logging level to use.
+   * @param replaceNewLines configures new lines replacement in the messages
    */
-  public static void configureDefaults(final String ident, final Level level) {
+  public static void configureDefaults(final String ident,
```

Comment:
This is a breaking change. Never remove methods unless doing a new major release.